firefox-ios/Client/Frontend/Browser/Tabs/Legacy/LegacyTabPeekPreviewActionBuilder.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+class LegacyTabPeekPreviewActionBuilder {
```

Comment:
This is another builder where a fluent API doesn't seem appropriate to me. If anyone disagrees, please let me know.